core/src/hash_table/file/mod.rs

```diff
@@ -61,7 +62,7 @@ impl FileTable {
 
     /// given a Table enum, ensure that the correct sub-directory exists and return the string path
     fn dir(&self, table: Table) -> Result<String, HolochainError> {
-        let dir_string = format!("{}/{}", self.path, table.to_string());
+        let dir_string = format!("{}{}{}", self.path, MAIN_SEPARATOR, table.to_string());
```

Comment:
This can be done much easier and cleaner (I love the path API):

First I'd make `self.path` a `Pathbuf` which is normally used for anything path related. Then you can use:

```rust
let dir = self.path.join(table.to_string());
```
This makes `dir` a `Pathbuf` which is an owned `Path`. I would rather return this than a `String`

Reply:
I can't do this because self.path is a String.
Can you write a ticket for this please?
The object of this PR is just to fix it so it works on windows.